test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -474,7 +474,7 @@ public async Task ExtractContextIrrespectiveOfTheFilterApplied()
         }
 
         [Fact]
-        public async Task BaggageClearedWhenActivityStopped()
+        public async Task BaggageIsNotClearedWhenActivityStopped()
```

Comment:
Not a blocking comment... but if the instrumentation had never had this behavior, then we likely would never had this test in the first place. Perhaps we just remove this test completely?